Ball.java

```diff
@@ -0,0 +1,10 @@
+package core.basesyntax;
+public class Ball {

```

Comment:
there always should be space between these sections: package - import - class 
use Intellij auto format - Ctrl Alt L for windows machines